app/src/test/java/org/mozilla/fenix/components/toolbar/DefaultBrowserToolbarControllerTest.kt

```diff
@@ -249,7 +249,7 @@ class DefaultBrowserToolbarControllerTest {
         controller.handleTabCounterItemInteraction(item)
         verify {
             homeViewModel.sessionToDelete = "1"
-            navController.navigate(BrowserFragmentDirections.actionGlobalHome())
+            navController.navigateBlockingForAsyncNavGraph(BrowserFragmentDirections.actionGlobalHome())
```

Comment:
I renamed `navigateBlockingForAsync` back to `navigate` for the test files and I don't get additional failures: why are we renaming all these instances in the tests? It's already a large change so I'd prefer if we didn't make all these changes unnecessarily.

The commands I ran to change these quickly were (make sure to commit before doing it):
```sh
find app/src/test -type f -exec sed -i "" "s/navigateBlockingForAsyncNavGraph/navigate/g" {} \;
find app/src/test -type f -exec sed -i '' "/import org.mozilla.fenix.ext.navigate/d" {} \;
```

You'll need to uncommit the change to the `*MockMaker` file too